src/components/FilterMenu/FilterMenu.js

```diff
@@ -136,6 +137,17 @@ const FilterMenu = ({
     }
   }
 
+  const onKeyDown = event => {
+    if (event.keyCode === 13) {
```

Comment:
We need to add common key names to **src/constants.js** file.
So we could have `ENTER_KEY` or `ENTER_KEY_CODE` or `KEY_CODES.ENTER` which has the value `13`.